api/scripts/getOldPolkadotRewardEras.js

```diff
@@ -119,13 +120,21 @@ function parseRewards(
             )
 
             staked = stakerExposure ? stakerExposure.value : ZERO
+            // console.log(`staked1:`, staked)
+            staked = isHex(staked) ? new BN(staked.substr(2, staked.length - 2), 16) : new BN(staked)
```

Comment:
that looks very hacky. why is staked a hex?

Reply:
Sometimes api returns either a integer or an hex, the old code was expecting a BN.

Reply:
see above: can we call a toHuman somewhere?